src/tram-one.ts

```diff
@@ -0,0 +1,17 @@
+export { registerHtml, registerSvg } from './dom-wrappers';
+export { default as useEffect } from './use-effect';
+export { default as useUrlParams } from './use-url-params';
+export { default as useStore } from './use-store';
+export { default as useGlobalStore } from './use-global-store';
+export { default as start } from './start';
+export {
+	ElementOrSelector,
+	DOMTaggedTemplateFunction,
+	TramOneComponent,
+	StoreObject,
+	CleanupEffect,
+	Effect,
+	Props,
+	Registry,
+	UrlMatchResults,
+} from './types';
```

Comment:
Exposing these types publicly, so that other Typescript projects can use them directly.